GETTING_STARTED_ADD_PAT_OWNER.md

```diff
@@ -0,0 +1,156 @@
+            * NOTE: If you already have a Cobalt template in mind, it may be a good idea to express that in the name.
+        * Select Create
+            * ![image](.Create_new_project.png)
+
+    * Create new repository by fetching source code from Cobalt's master project
+        * Select Repos tab within side-navigation menu
+        * Select 'Import a repository' from the Repos tab sub-menu
+        * Enter the Cobalt Clone URL (https://github.com/microsoft/cobalt.git) and select Import
+            * ![image](.Import-succesfull-image.png)
+
+    * Initialize new Azure Devops pipeline
+        * Select Pipelines tab from within side-navigation menu
+        * Select New pipeline and then choose 'Azure Repos Git [YAML]'
+        * Find and select the newly created repository from dropdown menu
+        * Import YAML by selecting 'Existing Azure Pipelines YAML file'
+            * Enter the path to the devops yaml file that lives within your newly created repo. (i.e. devops/providers/azure-devops/templates/azure-pipelines.yml)
+            * ![image](.Select-YAML-file.png)
+        * Review devops pipeline YAML and only keep templates relevant to your enterprise patterns.
+            * Remove jobName configurations not relevant to your enterprise patterns.
+                ```yaml
+                configurationMatrix:
+                - jobName: az_service_single_region
+                terraformTemplatePath: 'infra/templates/az-service-single-region'
+                terraformWorkspacePrefix: 'sr'
+                environmentsToTeardownAfterRelease:
+                - 'devint'
+                ```
+            * NOTE: If new to Cobalt, we recommend keeping the az_hello_world template as a starter template. This step can also be completed later as a code commit to your repo.
+        * Save and run
+            * NOTE: Azure Devops forces a run so expect this to fail. Future steps will solve this problem.
+
+2. Provision Azure resources needed for Azure Devops pipeline
+
+    * Create a Cobalt admin service-principal/service-endpoint for Cobalt deployments
+        * Sign-in to your organization's Azure account. (https://portal.azure.com)
+        * Filter for Azure Active Directory and navigate to it's menu
+        * Select App registrations from the menu blade
+        * Click [Add/+] New registration then enter a name for the application (ex. cobalt-hw-admin-sp)
+        * Choose single tenant as a supported account type to keep things simple.
+        * Click Register
+
+    * Configure the newly created registered app as a Service Principal
+        * From the app's service blade, select the Certificates & Secrets tab
+        * Click [Add/+] New from within Client secrets menu then enter a description (ex. rbac)
+        * Click Add
+            * NOTE: Take note of the generated client secret (only displayed once).
+        * From the app's service blade, select Overview. Take note of the Application (client) ID for the next step.
```

Comment:
This doesn't seem to be needed.

Reply:
Process has been clarified to show that it is used in creation of the Service Connection.